scss/_patterns_modal.scss

```
@@ -17,7 +17,7 @@
     right: 0;
     top: 0;
     width: 100%;
-    z-index: 101;
+    z-index: 104;
```

To be on a safe side and allow more wiggle room in future let's make this one higher:
```suggestion
    z-index: 150; // render on top of any other content or layout elements
```

---

Thanks for the quick review and for the suggestion! Done @ [5fde2d8](https://github.com/canonical/vanilla-framework/commit/5fde2d827e07e3913cb92609c3a5d3609b64b59e)